<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>531</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>536</width>
    <height>531</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>536</width>
    <height>531</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="loginInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordInput">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите логин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signInPushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signUpPushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
